front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java

```diff
@@ -469,17 +482,58 @@ public Void call() throws Exception {
         return objectMapper.readValue(json, clas);
       } catch (Exception ex) {
         if (current_version) {
-          log.error("Error reading {}: {}", value("object", object.getName()), ex);
+            log.error("Error reading {}: {}", value("object", object.getName()), ex.getMessage());
         } else {
           log.error("Error reading {} generation={}: {}",
                     value("object", object.getName()),
                     value("generation", object.getGeneration()),
-                    ex);
+                    ex.getMessage());
```

Comment:
Since we don't rethrow here, is it maybe valuable to print more than the message and include the stack trace?